stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -424,11 +432,18 @@ ::util::Status GetAllEntries(
     std::unique_ptr<::tdi::TableData> table_data;
     RETURN_IF_TDI_ERROR(table->keyAllocate(&table_key));
     RETURN_IF_TDI_ERROR(table->dataAllocate(&table_data));
-    RETURN_IF_TDI_ERROR(table->entryGetFirst(
-        *tdi_session, tdi_dev_target,
+    tdi_status = table->entryGetFirst(
+        *tdi_session, *dev_tgt,
         flags, table_key.get(),
-        table_data.get()));
-
+        table_data.get());
+    if (tdi_status != TDI_SUCCESS) {
+        //SDE iterates through all the tables, and if no entry is present for that table,
```

Comment:
Coding guidelines: Please wrap comment lines so they don't go past column 80.

Comments are also a little easier to read if you leave a space between `//` and the text.